kube/aks/scheduler-k8s.yaml

```diff
@@ -68,22 +92,6 @@ spec:
             - name: tmpsecrets
               mountPath: /home/kernelci/.azure
               subPath: k8s-credentials/.azure
-      initContainers:
```

Comment:
Please correct me if I'm wrong: this change is just a cleanup to group both `initContainers` together, not a functional one?

Reply:
Yes, i just spotted mistake when i was editing file. Not best approach to mix in PR, but at least did as separate commit. I was afraid yaml failure might lead to other failures, e.g. to read health section.